modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
+	return func(mci *service.ModuleCreateInfo) error {
+		mci.Items[_graphInterceptorKey] = graph
+		return nil
+	}
+}
+
+func graphFromCreateInfo(mci service.ModuleCreateInfo) dig.Graph {
+	g, ok := mci.Items[_graphInterceptorKey]
+	if !ok {
+		return dig.DefaultGraph()
+	}
+
+	// Intentionally panic if someone adds non-graph to Items.
+	return g.(dig.Graph)
```

Comment:
Hmm, how an explicit panic will be better than a regular one? It will provide all the types needed:
`panic: interface conversion: interface is *dig.graph, not string`